cmd/testConsumerProducer/main.go

```diff
@@ -35,7 +35,7 @@ func main() {
 		panic(err)
 	}
 
-	opt := func(config *gaz.ConsumerConfig) {
+	opt := func(config *gaz.StreamConsumerConfig) {
```

Comment:
EDIT: I don't care about this rename, it won't break my code. So go ahead

Would it be possible to introduce this PR without breaking the existing API?
That would make the migration simpler, otherwise it will be extremely painful to upgrade gorillaz.
I understand this config is specific to gorillaz stream, so having ConsumerConfig is not so gorillaz stream specific, but I'm afraid of the migration

What do you think?